test/src/analytics/testAnalytics.js

```diff
@@ -130,7 +130,7 @@ describe('test analytics utility', function() {
     var called = false;
 
     for (let i = 0; i < 7; i++) {
-      analytics = requireUncached('../../../lib/utils/analytics.js');
+      analytics = requireUncached('../../../dist/utils/analytics.js');
```

Comment:
I think one need to verify, but I feel that we can directly use `lib`

Reply:
This is because the tests should be run on the files which the users are going to use instead of running them on the files which are going to be omitted from out NPM package.